llms/mlx_lm/utils.py

```diff
@@ -35,6 +35,8 @@
     lambda m: isinstance(m, nn.Linear)
     and m.weight.shape[0]
     != 8  # avoid quantizing gate layers, otherwise we have to re-quant and upload all the mixtral models
+    and m.weight.shape[0]
+    not in [1, 60] # avoid quantizing qwen2_moe gate layers
```

Comment:
It's working fine without it but gimme a few minutes to test it.

Reply:
There is no difference in the response or speed so I think we can remove it.

Reply:
Great! Let me know when it's done then we can merge it!

Reply:
Remove it ✅

Reply:
I suggest not to quantizing the gating layers since the routing function is sensitive, unless you can prove that it won't affect downstream tasks at all. I did not quantize these layers in the GPTQ-int4 version.

Reply:
Would be great to see some results on that.. of course you always lose a little from quantizing any matrix but are the results more sensitive to quantizing the gate than other matrices?  That would be very useful to know if it's generally the case. 

(I believe @mzbac has possibly observed this as well)